.env.example

```diff
@@ -1,2 +1,7 @@
 ALGOLIA_API_KEY=
 ALGOLIA_BASE_URL=https://crawler.algolia.com
+
+# Netlify spec
+BRANCH="master"
+SITE_NAME="crawler-netlify-plugin"
+DEPLOY_PRIME_URL="https://master--crawler-netlify-plugin.netlify.app"
```

Comment:
I didn't set them in the `.env` because this is not part of our plugin configuration, it's inputs to the script.
But it's cleaner than having them in the source code indeed.
Maybe putting them in `run_dev` could be better.